src/Components/MovieTile/MovieTile.test.js

```diff
@@ -0,0 +1,57 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { MovieTile } from './MovieTile';
+import { NO_IMAGE_URL } from "../../Utils/constants";
```

Comment:
Take a look in aliases side to avoid following path: "../../"

Reply:
Could you provide more details here?